rmgpy/rmg/model.py

```diff
@@ -331,7 +323,20 @@ def makeNewSpecies(self, object, label='', reactive=True, checkForExisting=True)
         spec.molecularWeight = Quantity(spec.molecule[0].getMolecularWeight()*1000.,"amu")
         
         submit(spec)
-
+        
+        if spec.label == '' and spec.thermo and spec.thermo.label != '':
+            logging.info('Species with SMILES of {0} named {1} based on thermo library name'.format(molecule.toSMILES().replace('/','').replace('\\',''),spec.thermo.label))
+            spec.label = spec.thermo.label
+        
+        if label == '': 
```

Comment:
If we assign a value to `spec.label`, isn't it still possible for `label` to be empty (`''`)?